css/home.css

```diff
@@ -20,17 +13,20 @@
 }
 .reservation h3{
     text-align: center;
+    margin: auto 5%;
 }
-.location{
+.col{
```

Comment:
I would recommend not doing this and use https://getbootstrap.com/docs/5.0/utilities/text/#text-alignment instead